### ISSUE TITLE
Clarification on setup instructions

### ISSUE THREAD

**Issue by AlanBreck** [bennymi/presveltation#17]
Apologies if an issue isn't the best way to ask this, but I didn't see a Discussions tab.

This project looks fantastic. The feature that allows a separate, synced window for presenter notes is brilliant.

How do I scaffold a project out for this? The instructions refer to adding this as a dependency, but does it assume an existing SvelteKit project?

**Comment by bennymi** [comment]
@AlanBreck Yes that's correct, you have to use it with SvelteKit. I haven't updated this library in quite a while though, since I've been busy with other stuff, but will hopefully get some time soon again to upgrade it to Svelte 5. You can look at the docs demo if you need an example: https://github.com/bennymi/presveltation/tree/main/src/routes/(example)/example

If you're looking for presentation alternatives in Svelte, I think [animotion](https://animotion.pages.dev/docs) might also be interesting for you.
